<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomPayPerTimeDialog</class>
 <widget class="QDialog" name="CustomPayPerTimeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>CustomYesNoDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font: 600 9pt &quot;Segoe UI&quot;;
	border: none;
	background-color: #dfdfdf;
	padding: 0;
	margin: 0;
	color: #324b4c;
}

QDialog{
	background-color: rgba(255, 255, 255, 0);

}

QDateEdit::up-button, QDateEdit::down-button{
	width: 0px;
}


QLineEdit, QDateEdit{
	border: none;
	font-size: 9pt;
	border-bottom: 1px solid #7c9399;
}

QComboBox{
	background-color: #dfdfdf;
	border-bottom: 1px solid #7c9399;
	border-radius: 3px;
}

QComboBox::drop-down{
	image: url(':/icons/app/assets/icons/Alt-Arrow-Down--Streamline-Solar-Broken.svg');
	width: 16px;
	height: 16px;
}

QComboBox:hover{
	color: #2b2b2c;
	font: 700 9pt &quot;Segoe UI&quot;;
	border-radius: 0px;
	alternate-background-color: #2c313c;
}
QComboBox::item:selected{
	color: #2b2b2c;
	font: 700 9pt &quot;Segoe UI&quot;;
	border-radius: 0px;
}
QComboBox::item{
	color: #324b4c;
	font: 600 9pt &quot;Segoe UI&quot;;
	height: 10px;  
	border: 4px solid transparent;
	padding: 0px 4px 0px 4px;
}

#calStartBtn, #calLeavingBtn {
	background-color: #dfdfdf;
	padding: 0px;
}

#mainWidget{
	border-radius: 15px;
}

#widget * {
	font-size: 9pt;
}

#dialogTitle {
	font-size: 10pt;
	font-weight: 700;
}
#yesBtn {
	padding: 0px;
}

#yesBtn:hover{
	icon: url(:/icons/app/assets/icons/Check-Square--Streamline-Solar-Broken-#008b69.svg);
	color: #008b69;
	padding: 0px;
}


QScrollBar:vertical {
	width: 20px;
	background: none;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
	background: #dfdfdf;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QWidget" name="mainWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="dialogTitle">
           <property name="text">
            <string>Детайли за работник</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>290</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <property name="spacing">
            <number>5</number>
           </property>
           <item row="5" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Дата Активна</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Курс</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QWidget" name="widget_5" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="spacing">
               <number>10</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QDateEdit" name="activedDateDateEdit">
                <property name="focusPolicy">
                 <enum>Qt::ClickFocus</enum>
                </property>
                <property name="displayFormat">
                 <string>dd.MM.yy 'г.'</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignLeft">
               <widget class="QPushButton" name="calBtn">
                <property name="focusPolicy">
                 <enum>Qt::NoFocus</enum>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../../../resources.qrc">
                  <normaloff>:/icons/app/assets/icons/Calendar--Streamline-Solar-Broken.svg</normaloff>:/icons/app/assets/icons/Calendar--Streamline-Solar-Broken.svg</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>18</width>
                  <height>18</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Коментар</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="commentLineEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>€/Мин</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="levaPerMinLineEdit">
             <property name="locale">
              <locale language="English" country="UnitedStates"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Лв/Мин</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="euroPerMinLineEdit">
             <property name="locale">
              <locale language="English" country="UnitedStates"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>10</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLineEdit" name="euroPerLevaLineEdit">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="locale">
                 <locale language="English" country="UnitedStates"/>
                </property>
                <property name="text">
                 <string>1.9558</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_4" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item alignment="Qt::AlignRight">
                  <widget class="QCheckBox" name="percentageCheckBox">
                   <property name="text">
                    <string>%</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="percentageLineEdit">
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="placeholderText">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QWidget" name="buttonsWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>9</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="yesBtn">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>9</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string> ДА</string>
           </property>
           <property name="icon">
            <iconset resource="../../../resources.qrc">
             <normaloff>:/icons/app/assets/icons/Check-Square--Streamline-Solar-Broken.svg</normaloff>:/icons/app/assets/icons/Check-Square--Streamline-Solar-Broken.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
